README.md

```diff
@@ -59,6 +61,31 @@ Install specific files in Dom0:
 sudo qubesctl state.apply dotfiles.copy-dom0,dotfiles.copy-sh,dotfiles.copy-vim,dotfiles.copy-x11
 ```
 
+#### Pillar
+
+With Salt, the execution of each state can be opt out via the dotfiles pillar.
```

Comment:
... via the dotfiles pillar by using the state name without the `copy-` part as the key.

---

I don't find my explanation great at this stage either...

Reply:
would you prefer naming the pillar directly copy-*, eg: copy-dom0, so no further explanation required ?

Reply:
No, because I try to give meaningful names to the states and only having the program/project name, would be hard to understand what it does just from it's name.

The initial idea was, although not implemented, to have `install-vim.sls`, `install-tmux.sls`, states that install the necessary packages for each dotfile to work, but I didn't do that yet, so in the future, there would be `install-vim.sls` and `copy-vim.sls`, instead of `install-vim.sls` and `vim.sls`.

The reason for separation of the states is because the installation of programs only make sense on templates and standalones and I don't want the states to take more time than necessary.

So, I will try to find a better wording for this section.

Reply:
A way of doing it may be to create sub-directories for each topic so you could have dotfiles/dom0/install.sls and dotfiles/dom0/copy-dom0.sls (or rather simply dotfiles/dom0/config.sls), so you would not be tied to the name of the files but on the name of their directory.

Alternatively, do you think this phrase is better :?

> With salt, each state execution can be opt out by topic via a corresponding pillar set to a non true value (eg: Setting qusal:dotfiles:dom0 to false will disable states specific to dom0, setting qusal:dotfiles:git to false will disable states specific to git).